### DISCUSSION TITLE
Is there any workaround to make opacity/transparency setting work in Wayland?

### DISCUSSION THREAD

**Discussion by ljfp** [cpeditor/cpeditor#1412] [Q&A]
I recently migrated from a Desktop Environment running in X11 (GNOME) to another DE running in Wayland (COSMIC). It seems that CPEditor works fine in Wayland _except_ for the transparency. When I reduce the opacity setting, nothings changes. It remains at 100% opacity. I guess that's because X and Wayland handle transparency in a different way.
Is there any way to fix this?

**Comment by ouuan** [comment]
Is transparency working fine in other applications?

**Reply by ljfp** [reply]
My terminal emulator (Alacritty) has transparency set to 50% and it's working fine after the migration to Wayland.
